packages/eds-utils/src/hooks/useHideBodyScroll.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { render } from '@testing-library/react'
+import { useHideBodyScroll } from './useHideBodyScroll'
+
+const TestComponent = ({ hideScroll }) => {
+  useHideBodyScroll(hideScroll)
+  return <div></div>
+}
+
+describe('useHideBodyScroll', () => {
+  it('Should add "overflow: hidden" on document.body only while input is true', () => {
+    const { rerender } = render(<TestComponent hideScroll={true} />)
+    expect(document.body.style.overflow).toBe('hidden')
+    rerender(<TestComponent hideScroll={false} />)
+    expect(document.body.style.overflow).toBe('')
+  })
```

Comment:
Maybe add a test where body overflow is set to some other value and that its re-instated after `hideScroll` has been triggered?

Reply:
good point 👍

Reply:
Done